tests/unit/GoogleMapsFieldTest.php

```diff
@@ -50,17 +50,17 @@ public function testGetHiddenFields()
     {
         $field = GoogleMapsField::create('Test');
 
-        $fields = '<input type="hidden" name="GoogleMapsLatField" class="hidden" id="GoogleMapsLatField" />'
-            . '<input type="hidden" name="GoogleMapsLngField" class="hidden" id="GoogleMapsLngField" />'
-            . '<input type="hidden" name="subpremise" class="hidden" id="subpremise" />'
-            . '<input type="hidden" name="street_number" class="hidden" id="street_number" />'
-            . '<input type="hidden" name="route" class="hidden" id="route" />'
-            . '<input type="hidden" name="sublocality_level_1" class="hidden" id="sublocality_level_1" />'
-            . '<input type="hidden" name="locality" class="hidden" id="locality" />'
-            . '<input type="hidden" name="administrative_area_level_1" class="hidden"'
-            . ' id="administrative_area_level_1" />'
-            . '<input type="hidden" name="country" class="hidden" id="country" />'
-            . '<input type="hidden" name="postal_code" class="hidden" id="postal_code" />';
+        $fields = '<input type="hidden" name="TestGoogleMapsLatField" class="hidden" id="TestGoogleMapsLatField" />'
+            . '<input type="hidden" name="TestGoogleMapsLngField" class="hidden" id="TestGoogleMapsLngField" />'
+            . '<input type="hidden" name="Testsubpremise" class="hidden" id="Testsubpremise" />'
+            . '<input type="hidden" name="Teststreet_number" class="hidden" id="Teststreet_number" />'
+            . '<input type="hidden" name="Testroute" class="hidden" id="Testroute" />'
+            . '<input type="hidden" name="Testsublocality_level_1" class="hidden" id="Testsublocality_level_1" />'
+            . '<input type="hidden" name="Testlocality" class="hidden" id="Testlocality" />'
+            . '<input type="hidden" name="Testadministrative_area_level_1" class="hidden"'
+            . ' id="Testadministrative_area_level_1" />'
+            . '<input type="hidden" name="Testcountry" class="hidden" id="Testcountry" />'
+            . '<input type="hidden" name="Testpostal_code" class="hidden" id="Testpostal_code" />';
```

Comment:
These changes are completely irrelevant to what actually should be happening. The name should _not_ be a direct name but an array